### ISSUE TITLE
queue

### ISSUE THREAD

**Issue by zozo91** [barteksc/AndroidPdfViewer#404]
I need a full example please for using onDraw() method because it is not enough clear I tried to use it but without any benefit and I want congratulate this project developers because they add new features and method every time and I will try to contribute to develop with you as I can as possible.

**Comment by 1stmetro** [comment]
What exactly are you trying to do in onDraw () ?

**Comment by zozo91** [comment]
draw as s pen ?

**Comment by deep-mm** [comment]
Yes please, a detailed example explaining the usage of onDraw method will be very beneficial
